src/status_im2/contexts/wallet/home/view.cljs

```diff
@@ -46,12 +46,19 @@
    {:id :collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}
    {:id :activity :label (i18n/label :t/activity) :accessibility-label :activity-tab}])
 
-(defn view
+(defn f-view
   []
   (let [top          (safe-area/get-top)
         selected-tab (reagent/atom (:id (first tabs-data)))]
     (fn []
-      (let [networks (rf/sub [:wallet/network-details])]
+      (let [networks         (rf/sub [:wallet/network-details])
+            wallet-addresses (rf/sub [:wallet/all-addresses
+                                      networks (rf/sub [:wallet/network-details])])]
```

Comment:
Remove `(rf/sub [...])` in this line, it's being a second param not used by `all-addresses`